open_instruct/utils.py

```diff
@@ -51,6 +52,41 @@
 """
 
 
+# ----------------------------------------------------------------------------
+# Import utilities
+def check_hf_olmo_availability(return_version: bool = True) -> Union[dict, bool]:
+    pkg_name = "hf_olmo"
+
+    # Check if the package spec exists
+    package_exists = importlib.util.find_spec(pkg_name) is not None
+    package_version = "N/A"
+
+    if package_exists:
+        try:
+            # Primary method to get the package version
+            package_version = importlib.metadata.version(pkg_name)
```

Comment:
curious why doing this over like:
```python
try
    import hf_olmo
 except ImportError
    ...
```
is preferable. Don't mind it, just curious.